.github/workflows/build_and_test.yml

```diff
@@ -33,12 +33,17 @@ on:
     - cron: '0 7 * * *'
     # PySpark coverage for master branch
     - cron: '0 10 * * *'
+    # Java 11
+    - cron: '0 13 * * *'
+    # Java 17
+    - cron: '0 16 * * *'
```

Comment:
Unfortunately, we cannot test the daily jobs via PR Builder or [personal fork](https://github.com/dongjoon-hyun/spark/actions/workflows/build_and_test.yml?query=event%3Aschedule).